rs/nns/governance/src/governance.rs

```diff
@@ -2444,7 +2454,7 @@ impl Governance {
             command: Some(InFlightCommand::Split(*split)),
         };
 
-        let staked_amount = split_amount_e8s - transaction_fee_e8s;
+        let staked_amount = split.amount_e8s.saturating_sub(transaction_fee_e8s);
```

Comment:
Here is an example of the other case where I think saturating makes sense, i.e. there is some nearby code that ensures that this will not overflow. To wit, line 2407 ensures that split.amount_e8s - transaction_fee_e8s does not overflow. In these kinds of cases, I think it is good to have a comment that points back to the code that prevents overflow here.